packages/components/src/internal/components/entities/FindDerivativesButton.tsx

```diff
@@ -68,6 +72,34 @@ export const getSessionSearchFilterProps = (
     // always include viewFilters and user defined filters (filterArray)
     fieldFilters = fieldFilters.concat(filters.map(filter => getFieldFilter(model, filter)));
 
+    // Issue 47087: if model has selections, include those as IN clause
+    if (model.hasSelections && model.queryInfo.pkCols?.length === 1 && model.queryInfo.titleColumn) {
```

Comment:
I'm curious when `titleColumn` would not be defined. Is it a real case or a "just in case" case?

Reply:
Just in case. I imagine that titleColumn is always defined. I don't know of a case where it is not.